File: hackfsu/SponsorsViewController.m
//
//  SponsorsViewController.m
//  hackfsu
//
//  Created by Logan Isitt on 3/21/14.
//  Copyright (c) 2014 Isitt Inc. All rights reserved.
//

#import "SponsorsViewController.h"

@implementation SponsorsViewController

-(id)initWithCoder:(NSCoder *)aDecoder
{
    self = [super initWithCoder:aDecoder];
    if (self)
    {
        self.title = @"Sponsors";
        self.screenName = self.title;
    }
    return self;
}

- (void)viewDidLoad
{
    [super viewDidLoad];
}

@end

File: HackFSU/SponsorsViewController.swift
//
//  SponsorsViewController.swift
//  HackFSU
//
//  Created by Logan Isitt on 1/31/15.
//  Copyright (c) 2015 Logan Isitt. All rights reserved.
//

import UIKit

class SponsorsViewController: UITableViewController {
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        self.setNavigationBarItem()
        
        self.title = "Sponsors"
        self.navigationItem.title = self.title
    }
}